src/main/kotlin/car/domain/Winners.kt

```diff
@@ -0,0 +1,19 @@
+package car.domain
+
+class Winners(carMoveHistoryCollection: CarMoveHistoryCollection) {
+    val winners: List<Winner>
+
+    init {
+        val maxPosition = carMoveHistoryCollection.moveHistories
+            .map { it.position }
+            .max()!!
+
+        winners = carMoveHistoryCollection.moveHistories
+            .filter { it.isWinner(maxPosition) }
+            .map { Winner(it.carName) }
+    }
+
+    override fun toString(): String {
+        return winners.joinToString(",") { toString() }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return winners.joinToString(",")
```